template-visual/meta_data.json

```diff
@@ -1,5 +1,4 @@
 {
-    "_comment": "The description and title fields will be edited to fit your story. This _comment property can can be removed or left as it is, as it will not be included in the fields sent to the server.",
```

Comment:
This line is never kept, so I'm just deleting it.

Reply:
was going to suggest the same, thanks!